src/opm/output/eclipse/AggregateWellData.cpp

```diff
@@ -239,7 +237,7 @@ namespace {
             return WMCtrlVal::WMCtlUnk;
         }
 
-        int compOrder(const Opm::Well2& well)
+      int compOrder(const Opm::Well2& well, const Opm::SummaryState& st)
```

Comment:
Why does the static connection ordering need a dynamic `SummaryState`?  As far as I can tell, the `st` parameter is unused in `compOrder()`.

Also, please restore the original indentation level of the function declaration.

Reply:
> Why does the static connection ordering need a dynamic SummaryState? As far as I can tell, the st parameter is unused in compOrder().

That is right - removed now.